src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -28,6 +28,17 @@
         line-height: 20px;
     }
 
+    &.bcs-header-shadow::before {
+        position: absolute;
+        // position it behind bcs-content-header
+        top: -60px;
+        right: 20px;
+        left: 20px;
+        height: 60px;
+        box-shadow: 0 4px 12px rgba(0, 0, 0, .1);
+        content: ' ';
+    }
```

Comment:
could you add a screenshot of this?

Reply:
where did the 20px and 60px values come from?

Reply:
60px is the height of the Content header, it's defined in `bsc-content-header` class in https://github.com/box/box-ui-elements/blob/8d2c3f117a5ff26993ae0b78c7b8157091b18ad7/src/elements/content-sidebar/SidebarContent.scss#L13

Reply:
I've added screenshot of the design in the PR description.